pkg/server/plugin/datastore/sql/migration.go

```diff
@@ -131,3 +133,12 @@ func migrateToV1(tx *gorm.DB) error {
 	}
 	return nil
 }
+
+func migrateToV2(tx *gorm.DB) error {
+	// add the federated_with column
+	if err := tx.AutoMigrate(&RegisteredEntry{}).Error; err != nil {
```

Comment:
I guess it depends on "work as expected" :) It will bring the table up to the latest schema, which could possible break migrations that come after that (that hopefully the unit tests would catch). I was hoping GORM exported AddColumn functionality, but it does not. If needed, we could crib some code from GORM (all of the code that does the "add column" is exported, i think).

Reply:
Heh... I was thinking to use `tx.Exec` again as we did in migrateToV1? Is that a bad idea?

Reply:
Aah, right on